# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoderStateMachine.cpp
# Commit: 16633fa96586
# Full Hash: 16633fa96586cb01af6dbfb03d062f1b32691ae9
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-05-24 21:44:48
# Regressor Bug: 1743834
# File Overlap Count: 1
# Description:
#   Bug 1743834 - Split initialization and start in AudioSinkWrapper. r=alwu
#   
#   No change in functionality, this is to prepare for the next patch that will do
#   the initialization asynchronously because it's expensing and blocking.
#   
# ==============================================================================

diff -r 505d7df63978 -r 16633fa96586 dom/media/MediaDecoderStateMachine.cpp
--- a/dom/media/MediaDecoderStateMachine.cpp	Tue May 24 13:09:09 2022 +0000
+++ b/dom/media/MediaDecoderStateMachine.cpp	Tue May 24 13:09:09 2022 +0000
@@ -2855,10 +2855,10 @@
   }
 
   RefPtr<MediaDecoderStateMachine> self = this;
-  auto audioSinkCreator = [self](const media::TimeUnit& aStartTime) {
+  auto audioSinkCreator = [self]() {
     MOZ_ASSERT(self->OnTaskQueue());
     AudioSink* audioSink =
-        new AudioSink(self->mTaskQueue, self->mAudioQueue, aStartTime,
+        new AudioSink(self->mTaskQueue, self->mAudioQueue,
                       self->Info().mAudio, self->mSinkDevice.Ref());
     self->mAudibleListener.DisconnectIfExists();
     self->mAudibleListener = audioSink->AudibleEvent().Connect(